kbhff/test/login_test.py

```diff
@@ -129,3 +129,19 @@ def test_unverifiedUserWithPasswordFirstLoginDuplicateVerificationTab(driver, gm
     assert_current_page_is("login", driver)
     assert_username_prefilled(user["email"], driver)
     assert_text_on_page("Din konto er allerede aktiveret!", driver)
+
+#Test specification number 7 is a duplicate of number 5

```

Comment:
Not quite: as I read it, test 7 is two tests in one, one of which is a duplicate of number 5. The "unimplemented" test is when the unverified user is signed up via medlemshjælp. I seem to remember Søren confirming that this was a mistake and saying he would fix it in the spreadsheet... (On Slack.)